test/kokoro/xds.sh

```diff
@@ -13,7 +13,13 @@ shopt -s extglob
 branch="${branch//[[:space:]]}"
 branch="${branch##remotes/origin/}"
 shopt -u extglob
-go build
+# Install go >= 1.14 for the new features, e.g. errors.Is()
```

Comment:
This comment will be outdated eventually.

How about "Install a version of Go supported by gRPC"?